docs/ibc-relayer.md

```diff
@@ -0,0 +1,78 @@
+when relaying with Babylon, particularly around its unique unbonding period
+mechanism.
+
+## Important Note on Babylon's Unbonding Period
+
+Babylon has a unique unbonding mechanism that differs from standard Cosmos SDK chains. The Babylon Genesis chain disables the standard `x/staking` module and wraps it with the `x/epoching` module, introducing secure, fast unbonding through Bitcoin timestamping.
+
+**Important**: The standard `x/staking` module's unbonding time parameter remains at the default 21 days, but **this value should be ignored** when configuring the relayer's trusting period.
+
+1. **Epoching System**:
+   - All staking operations and voting power adjustments are processed at the final block of each epoch
+   - The final block of each epoch is checkpointed onto the Bitcoin blockchain
+   - Each epoch spans 360 blocks (defined by `epoch_interval` parameter of [x/epoching module](https://github.com/babylonlabs-io/babylon/blob/main/x/epoching/README.md))
+   - With 10s block times, each epoch duration is 1 hour
+
+2. **Finalization Process**:
+   - After an epoch is timestamped on a Bitcoin block, it becomes finalized once the block is 300-deep
+   - This is defined by the `checkpoint_finalization_timeout` parameter of [x/btccheckpoint module](https://github.com/babylonlabs-io/babylon/blob/main/x/btccheckpoint/README.md)
+   - Any unbonding requests from that checkpointed epoch are then matured
+   - Given Bitcoin's average block time of ~10 minutes, the average unbonding time is about 50 hours
+
+3. **IBC Light Client Configuration**:
+   - IBC light clients for Babylon Genesis on other chains should have a lower trusting period (~33 hours)
+   - This is about 2/3 of the unbonding period, following standard IBC security practices
+   - This configuration only affects light clients of Babylon Genesis on other chains
+   - The trusting period of other chains' light clients on Babylon Genesis remains unaffected
+
+Due to these unique characteristics, special attention is required when configuring the relayer's trusting period and client refresh rate.
+
+## Prerequisites
+
+Before beginning, ensure you have:
+1. Rust installed and configured
+2. Hermes installed (refer to [Hermes Quick Start](https://hermes.informal.systems/quick-start/) for installation steps)
+3. Access to RPC and gRPC endpoints for both chains
+4. Wallets funded with native tokens for both chains
+
+## Relayer Configuration
+
+When setting up a relayer for Babylon, pay special attention to these parameters:
+
+1. **Trusting Period**: Should be set to approximately 2/3 of Babylon's unbonding period
+   - Babylon's unbonding period is ~50 hours (based on ~300 BTC blocks)
+   - Therefore, the trusting period should be set to ~33 hours
+
+2. **Client Refresh Rate**: A higher refresh rate is recommended (1/5 of trusting period, i.e., ~6.6 hours)
+   - Make sure `refresh = true` is set in the configuration
```

Comment:
```suggestion
   - For example, for hermes relayer operators, the configuration `refresh = true` is set
```

Let's make the guide general and not assume that someone is running hermes. We can always use hermes as an example though to describe the values.